VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -750,26 +752,27 @@ private void clearAllErrors() {
 
     @Override
     public void onPrepared() {
-        if ( !isResumed() 
-                || !isVisible()) {
+        // mark prepared and allow orientation
+        isPrepared = true;
+
+        if (!isResumed() ||
+                (getParentFragment() != null && !getParentFragment().getUserVisibleHint())) {
             freezePlayer();
             return;
         }
 
+        environment.getSegment().trackVideoLoading(videoEntry.videoId, videoEntry.eid,
+                videoEntry.lmsUrl);
```

Comment:
Why move this before `clearAllErrors()`? The tracking call should probably be performed at the end.